docs/dockerfile_experimental.md

```diff
@@ -0,0 +1,128 @@
+# Dockerfile frontend experimental syntaxes
+
+The contents of this document is planned to be moved to https://docs.docker.com in future.
```

Comment:
Is this information useful to the reader of this doc? Perhaps it should be an issue on some relevant repo?

Reply:
deleted